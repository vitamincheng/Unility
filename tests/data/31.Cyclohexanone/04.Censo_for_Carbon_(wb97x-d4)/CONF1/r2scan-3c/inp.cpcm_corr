Corrected dielectric energy   =     -0.008252682
Total C-PCM charge            =     -0.000000000
C-PCM corrected charges:
    -0.000000014
     0.000245544
    -0.000018657
    -0.000017719
    -0.000000013
    -0.000000018
    -0.000147374
    -0.000217376
     0.000931328
     0.000931525
    -0.000280212
    -0.000280230
     0.000852670
     0.000854062
     0.000104445
     0.000104619
     0.001046811
     0.001047059
    -0.000002860
    -0.000002861
    -0.000036036
    -0.000035115
    -0.000000488
    -0.000000488
     0.000386637
    -0.000000491
     0.000389190
    -0.000000491
     0.000424392
     0.000424316
    -0.000000011
    -0.000095561
    -0.000000011
    -0.000095776
    -0.000000400
    -0.000000400
    -0.000000045
    -0.000000045
     0.001145242
     0.001145374
    -0.000146239
    -0.000146245
    -0.000105476
    -0.000111094
    -0.000105812
    -0.000110940
    -0.000004220
    -0.000004217
     0.000797507
     0.000798017
    -0.000008083
    -0.000008090
     0.001283635
     0.001284052
     0.000527436
    -0.000136958
     0.000527403
    -0.000136822
    -0.000000002
    -0.000000002
    -0.000000027
    -0.000000027
    -0.000114772
    -0.000114764
    -0.000144354
    -0.000144370
     0.000039682
    -0.000260115
    -0.000047350
     0.001283290
    -0.000003073
    -0.000003074
    -0.000000326
    -0.000000329
    -0.000027548
    -0.000027526
    -0.000000457
    -0.000000457
    -0.000089208
    -0.000257174
    -0.000089607
    -0.000256837
     0.001347291
     0.001347649
    -0.000208596
    -0.000208598
     0.000938074
     0.000938254
    -0.000069327
    -0.000069340
    -0.000001144
    -0.000607757
    -0.000001148
    -0.000607363
    -0.000018400
    -0.000018347
     0.000265255
     0.000265483
    -0.000000005
    -0.000035525
    -0.000012818
    -0.000081176
     0.000785397
    -0.000159556
     0.000918926
    -0.000420643
    -0.000001465
    -0.000000090
    -0.000853531
     0.001273274
     0.000425763
    -0.000001752
    -0.000187816
    -0.000091940
    -0.000038229
    -0.001322129
     0.001090528
    -0.000362680
    -0.000169104
    -0.000000761
    -0.000626254
     0.001411705
     0.000346811
    -0.000025003
    -0.000000020
    -0.000065186
    -0.000000053
    -0.000037196
    -0.000000012
     0.000653368
    -0.000147131
     0.000787800
    -0.000075140
     0.000746150
    -0.000251063
    -0.000090986
    -0.000234965
     0.000739233
    -0.000133247
    -0.000267534
    -0.000000055
     0.000306011
     0.000736496
    -0.000000004
     0.000864455
    -0.000032178
    -0.000031506
    -0.000238700
     0.000144927
    -0.000067069
     0.000331593
    -0.000000441
    -0.000189986
     0.000348388
     0.000020671
    -0.000000012
     0.000454835
    -0.000051776
    -0.000017698
    -0.000057969
     0.000082973
    -0.000000052
    -0.000000368
    -0.000000078
    -0.000002522
    -0.000105723
     0.000756364
    -0.000061238
     0.001299087
     0.001102523
    -0.000006143
    -0.000208591
    -0.000304219
    -0.000143847
    -0.000175218
    -0.000063867
    -0.000000015
     0.000182941
    -0.000003854
    -0.000059938
    -0.000008754
     0.000914775
    -0.000024468
    -0.000066117
    -0.000201786
    -0.000006004
     0.000031882
    -0.000005933
     0.000064137
    -0.000003667
     0.000503811
    -0.000001358
    -0.000026359
    -0.000000768
    -0.000144821
    -0.000013539
    -0.000310616
    -0.000100006
     0.000075486
    -0.000441071
    -0.000132728
    -0.000000114
    -0.000000284
    -0.000226633
     0.000231602
     0.000980362
    -0.000095803
    -0.000259695
    -0.000116984
    -0.000827537
    -0.000163920
    -0.000484164
     0.000666082
    -0.000000066
    -0.000012850
    -0.000931230
    -0.000000075
    -0.000174493
     0.000876422
     0.000084879
    -0.000000207
    -0.000115269
    -0.000000004
    -0.000026632
    -0.000040011
    -0.000000002
     0.000674072
    -0.000238077
     0.000000350
     0.000142353
    -0.000332598
    -0.000356138
    -0.000455732
    -0.000109504
     0.000112847
    -0.000000019
    -0.000000071
     0.000639535
     0.000381441
    -0.000006632
    -0.000034004
     0.000585363
    -0.000956614
    -0.000201643
    -0.000000001
    -0.000123822
    -0.000004023
     0.000372964
    -0.000185421
     0.000000119
     0.000487365
    -0.000000147
    -0.000128283
    -0.000238662
    -0.000019707
    -0.000001195
    -0.000046499
    -0.000109596
    -0.000001299
    -0.000000022
    -0.000129333
     0.000684816
    -0.000235365
    -0.000007897
     0.000792375
     0.001022801
    -0.000000078
    -0.000006518
    -0.000131967
    -0.000308695
    -0.000715172
    -0.000037969
    -0.000197237
    -0.000001523
    -0.000081517
    -0.000021322
     0.000177251
    -0.000002162
    -0.000231160
     0.000006577
    -0.000011075
    -0.000788579
    -0.000003315
    -0.000359951
    -0.000001703
     0.000074745
    -0.000005605
    -0.000000056
    -0.000473338
    -0.000528555
    -0.000683190
    -0.000389109
    -0.000718941
    -0.000876416
    -0.000948325
    -0.000881391
    -0.000961692
    -0.000005870
    -0.000079481
    -0.000019733
    -0.000065814
    -0.000000192
    -0.000533779
    -0.000000221
    -0.000438000
    -0.000001030
    -0.000455219
    -0.000007651
    -0.000332556
    -0.000608863
    -0.000920268
    -0.000567815
    -0.000934223
    -0.000632099
    -0.000679607
    -0.000606059
    -0.000756094
    -0.000782381
    -0.000952008
    -0.000770944
    -0.000996810
    -0.000274280
    -0.000578087
    -0.000057855
    -0.000483690
    -0.000170849
    -0.000721884
    -0.000137992
    -0.000630188
    -0.000002688
    -0.000000002
    -0.000003242
    -0.000013281
    -0.000014037
    -0.000854721
    -0.001096634
    -0.000308310
    -0.000757438
    -0.000925790
    -0.000961792
    -0.000600497
    -0.000396624
    -0.000235217
    -0.000369550
    -0.000049874
    -0.000540398
    -0.000000385
    -0.001068408
    -0.000385574
    -0.000000012
    -0.000028928
    -0.000000017
    -0.000464265
    -0.000265454
    -0.000757228
    -0.000536807
    -0.000898085
    -0.000674340
    -0.000177006
    -0.000507995
    -0.000335240
    -0.000623959
    -0.000964614
    -0.000028383
    -0.001155906
    -0.000212006
    -0.000000122
    -0.000212560
    -0.000367674
    -0.000786924
    -0.000342133
    -0.000121214
    -0.000995470
    -0.000736945
    -0.000129091
    -0.000000000
    -0.000002618
    -0.000797994
    -0.000339023
    -0.000781748
    -0.000344766
    -0.001133113
    -0.000643842
    -0.000000080
    -0.000613452
    -0.000052160
    -0.000948489
    -0.001099409
    -0.000499616
    -0.000000169
    -0.000929058
    -0.000000000
    -0.000000200
    -0.000486314
    -0.000000001
    -0.000213105
    -0.000637176
    -0.000013006
    -0.000687420
    -0.000506827
    -0.000953198
    -0.000408173
    -0.000008858
    -0.000008864
    -0.000029808
    -0.000001524
     0.000000074
     0.000000443
     0.000000444
     0.000000216
     0.000000216
    -0.000000008
    -0.000000008
    -0.000003135
    -0.000003139
    -0.001027575
    -0.001027709
    -0.001189779
    -0.001189873
    -0.000016499
    -0.000016499
    -0.000617053
    -0.000617096
    -0.000000912
    -0.000000914
    -0.000021979
    -0.000021979
    -0.001052317
    -0.000938722
    -0.001052467
    -0.000938780
    -0.001084407
    -0.001469350
    -0.001081252
     0.000000305
     0.000000305
    -0.000056102
    -0.000056080
    -0.000000012
    -0.000000012
    -0.000373508
     0.000000001
    -0.000373594
     0.000000001
    -0.000006792
    -0.000006796
    -0.001596542
    -0.001596434
    -0.000000005
    -0.000277511
    -0.000000005
    -0.000277516
     0.000000001
     0.000000001
    -0.000437105
    -0.000433584
    -0.000147968
    -0.000427694
    -0.001341053
    -0.000000042
    -0.000738989
    -0.000000132
    -0.000120413
    -0.000387682
    -0.000720800
    -0.000000003
    -0.000808871
    -0.001052009
    -0.000661854
    -0.000889646
     0.000012258
    -0.000891533
    -0.000093932
    -0.000784614
    -0.000172162
    -0.001402545
    -0.000010366
    -0.001177069
    -0.000012880
    -0.001139286
    -0.000000165
    -0.000359922
    -0.000956881
    -0.001239322
    -0.000093963
    -0.000536688
    -0.000000780
    -0.000433280
    -0.001090257
    -0.000184770
    -0.000819519
    -0.001351736
    -0.000316315
    -0.000945496
    -0.001239276
    -0.000000007
    -0.000850177
    -0.000661311
    -0.001294931
    -0.001188377
    -0.000000006
    -0.000922690
    -0.000000130
    -0.001161089
    -0.000400934
    -0.000941617
    -0.000009397
    -0.001147470
    -0.000655577
    -0.000000043
    -0.000000000
    -0.000603267
    -0.000796960
    -0.001164940
    -0.001050493
    -0.000862473
    -0.001182758
    -0.001144587
    -0.001227392
    -0.001175054
    -0.000063326
    -0.000000876
    -0.000173021
    -0.000059555
    -0.000416160
    -0.000000361
    -0.000494407
    -0.000000581
    -0.000000000
    -0.000345997
    -0.000000235
    -0.000588303
    -0.000024925
    -0.001076490
    -0.001072938
    -0.001186527
    -0.001003395
    -0.000941111
    -0.001034251
    -0.001145400
    -0.001045219
    -0.001167848
    -0.001127397
    -0.001311830
    -0.001160123
    -0.000748610
    -0.000482134
    -0.000865222
    -0.000473270
    -0.000757554
    -0.000353286
    -0.000865528
    -0.000324369
    -0.000000864
    -0.000009654
    -0.000000023
    -0.000003684
    -0.000023363
    -0.000000000
    -0.001352278
    -0.001217822
    -0.000880330
    -0.000621478
    -0.001211946
    -0.001328243
    -0.001039310
    -0.000859504
    -0.000144843
    -0.000013594
    -0.000310497
     0.000075292
    -0.000099864
    -0.000441052
    -0.000132421
    -0.000000114
    -0.000000284
     0.000231341
     0.000980508
    -0.000226746
    -0.000095704
    -0.000259702
    -0.000116948
    -0.000484320
     0.000666275
    -0.000828216
    -0.000163899
    -0.000000066
    -0.000012836
    -0.000174820
     0.000876485
    -0.000930814
    -0.000000075
     0.000085038
    -0.000000004
    -0.000026645
    -0.000000207
    -0.000115404
    -0.000039835
    -0.000000002
     0.000674143
    -0.000238069
     0.000000350
     0.000142430
    -0.000332481
    -0.000109371
     0.000113080
    -0.000356189
    -0.000455552
    -0.000000019
     0.000639587
     0.000381675
    -0.000000071
    -0.000006625
    -0.000033962
     0.000585505
    -0.000201056
    -0.000956310
    -0.000000001
    -0.000123748
    -0.000004023
     0.000373054
    -0.000185459
     0.000000119
     0.000487547
    -0.000000147
    -0.000128288
    -0.000238568
    -0.000019735
    -0.000001193
    -0.000109354
    -0.000046535
    -0.000000022
    -0.000001302
    -0.000129322
     0.000684907
    -0.000235244
    -0.000007891
     0.000792341
     0.001022873
    -0.000006523
    -0.000131820
    -0.000000078
    -0.000037923
    -0.000197201
    -0.000308844
    -0.000715418
    -0.000021303
     0.000177385
    -0.000001521
    -0.000081501
    -0.000002161
    -0.000231177
     0.000006613
    -0.000003308
    -0.000359692
    -0.000011066
    -0.000788377
    -0.000001704
     0.000074973
    -0.000005609
    -0.000000005
    -0.000035519
    -0.000012862
     0.000786257
    -0.000081137
    -0.000158938
     0.000918992
    -0.000420627
    -0.000001467
    -0.000000090
     0.001273087
     0.000426694
    -0.000853260
    -0.000001755
    -0.000187770
    -0.000091865
     0.001090636
    -0.000362069
    -0.000038239
    -0.001321892
    -0.000168959
     0.001412001
     0.000347886
    -0.000000760
    -0.000626092
    -0.000025064
    -0.000000020
    -0.000037162
    -0.000000012
    -0.000065039
    -0.000000053
     0.000653524
    -0.000147044
     0.000788062
    -0.000074867
     0.000745840
    -0.000251242
     0.000739442
    -0.000133357
    -0.000091173
    -0.000234828
    -0.000267342
     0.000305094
     0.000736944
    -0.000000055
    -0.000000004
     0.000864009
    -0.000032129
    -0.000031510
     0.000144661
    -0.000239326
    -0.000067012
     0.000331696
    -0.000000442
    -0.000189794
     0.000348175
     0.000020578
    -0.000000012
     0.000455117
    -0.000052266
    -0.000017654
    -0.000058761
     0.000082846
    -0.000000052
    -0.000000078
    -0.000000366
    -0.000105813
    -0.000002523
     0.000756300
    -0.000061311
     0.001299092
     0.001102903
    -0.000006160
    -0.000208314
    -0.000175151
    -0.000063924
    -0.000304099
    -0.000143668
     0.000183709
    -0.000003860
    -0.000000015
     0.000914825
    -0.000024427
    -0.000059948
    -0.000008739
    -0.000066195
    -0.000201664
    -0.000005992
     0.000063966
    -0.000003674
     0.000031484
    -0.000005934
     0.000503357
    -0.000001353
    -0.000026264
    -0.000000768
    -0.000437088
    -0.000433605
    -0.000148083
    -0.000427741
    -0.001341147
    -0.000000042
    -0.000739014
    -0.000387778
    -0.000720838
    -0.000000132
    -0.000120512
    -0.000000003
    -0.000808912
    -0.001052064
    -0.000661882
    -0.000889685
    -0.000891628
     0.000012257
    -0.000784655
    -0.000093945
    -0.000172183
    -0.001402625
    -0.000010365
    -0.001177124
    -0.001139346
    -0.000012830
    -0.000359908
    -0.000000165
    -0.000956902
    -0.001239409
    -0.000093946
    -0.000536709
    -0.000433331
    -0.001090349
    -0.000000781
    -0.000184749
    -0.000819537
    -0.001351815
    -0.000316296
    -0.000945532
    -0.001239364
    -0.000850214
    -0.000000007
    -0.000661341
    -0.001294994
    -0.001188446
    -0.000000006
    -0.000922688
    -0.000000129
    -0.001161062
    -0.000400950
    -0.000941654
    -0.000009394
    -0.001147538
    -0.000655601
    -0.000000043
    -0.000000000
    -0.000603255
    -0.001050623
    -0.000862378
    -0.000797028
    -0.001164680
    -0.001227406
    -0.001175017
    -0.001182748
    -0.001144524
    -0.000173136
    -0.000059589
    -0.000063285
    -0.000000874
    -0.000494794
    -0.000000580
    -0.000416562
    -0.000000360
    -0.000000000
    -0.000588639
    -0.000024921
    -0.000346120
    -0.000000234
    -0.001186653
    -0.001003265
    -0.001076590
    -0.001072775
    -0.001145474
    -0.001045210
    -0.000941086
    -0.001034173
    -0.001311902
    -0.001160058
    -0.001167860
    -0.001127273
    -0.000865311
    -0.000473288
    -0.000748639
    -0.000481828
    -0.000865658
    -0.000324264
    -0.000757799
    -0.000352990
    -0.000009672
    -0.000000023
    -0.000000864
    -0.000023399
    -0.000000000
    -0.000003686
    -0.001352333
    -0.001217708
    -0.000880522
    -0.000621228
    -0.001328254
    -0.001211883
    -0.000859541
    -0.001039182
    -0.000000056
    -0.000473243
    -0.000388830
    -0.000718405
    -0.000528622
    -0.000682918
    -0.000881194
    -0.000961436
    -0.000876285
    -0.000948132
    -0.000019709
    -0.000065713
    -0.000005886
    -0.000079623
    -0.000000221
    -0.000437558
    -0.000000193
    -0.000533479
    -0.000007655
    -0.000332022
    -0.000001034
    -0.000455231
    -0.000567620
    -0.000933774
    -0.000608758
    -0.000919913
    -0.000605684
    -0.000755632
    -0.000632038
    -0.000679471
    -0.000770705
    -0.000996416
    -0.000782318
    -0.000951779
    -0.000057592
    -0.000483103
    -0.000274562
    -0.000577997
    -0.000137989
    -0.000629675
    -0.000171030
    -0.000721475
    -0.000000002
    -0.000003233
    -0.000002694
    -0.000014008
    -0.000013285
    -0.000854585
    -0.001096307
    -0.000308329
    -0.000756957
    -0.000961457
    -0.000925662
    -0.000396098
    -0.000600567
    -0.000235270
    -0.000369545
    -0.000049759
    -0.000540644
    -0.000000385
    -0.001068345
    -0.000385217
    -0.000000012
    -0.000464075
    -0.000265188
    -0.000028877
    -0.000000017
    -0.000757288
    -0.000536794
    -0.000898065
    -0.000674249
    -0.000508162
    -0.000177077
    -0.000624051
    -0.000335348
    -0.000964785
    -0.000028376
    -0.001155958
    -0.000211794
    -0.000212946
    -0.000000123
    -0.000786847
    -0.000367719
    -0.000342300
    -0.000121307
    -0.000995307
    -0.000736670
    -0.000129224
    -0.000000000
    -0.000797824
    -0.000338834
    -0.000002614
    -0.000781910
    -0.000344772
    -0.001133072
    -0.000643572
    -0.000613703
    -0.000000081
    -0.000948516
    -0.000052186
    -0.001099481
    -0.000499483
    -0.000929201
    -0.000000000
    -0.000000169
    -0.000000201
    -0.000486191
    -0.000000001
    -0.000213468
    -0.000637197
    -0.000013030
    -0.000687148
    -0.000506935
    -0.000953057
    -0.000000751
    -0.000421796
    -0.000567259
    -0.000399245
    -0.000566628
    -0.000398284
    -0.000865777
    -0.000751708
    -0.000865636
    -0.000751560
    -0.000068140
    -0.000007145
    -0.000067740
    -0.000007088
    -0.000000006
    -0.000000006
    -0.000059841
    -0.000000038
    -0.000059655
    -0.000000038
    -0.000000001
    -0.000000001
    -0.000189373
    -0.000000470
    -0.000188597
    -0.000000468
    -0.000762227
    -0.000325259
    -0.000762008
    -0.000324962
    -0.000670441
    -0.000544827
    -0.000669708
    -0.000544069
    -0.000882433
    -0.000596598
    -0.000882064
    -0.000596186
    -0.000392719
    -0.000214529
    -0.000391717
    -0.000213673
    -0.000317165
    -0.000160920
    -0.000316830
    -0.000160546
    -0.000002911
    -0.000002885
    -0.000007801
    -0.000007768
    -0.000981514
    -0.000618068
    -0.000414679
    -0.000232535
    -0.000848010
    -0.000847547
    -0.000413619
    -0.000412325
    -0.000375881
    -0.000213970
    -0.000814148
    -0.000814147
    -0.000059097
    -0.000059065
    -0.000497938
    -0.000189113
    -0.000497947
    -0.000189112
    -0.000118909
    -0.000118875
    -0.000000837
    -0.000000836
    -0.000847737
    -0.000847732
    -0.000591734
    -0.000591727
    -0.000700174
    -0.000700160
    -0.000366811
    -0.000366761
    -0.001056235
    -0.001056234
    -0.000029054
    -0.000029039
    -0.000519595
    -0.000006352
    -0.000519638
    -0.000006350
    -0.000745236
    -0.000000484
    -0.000745245
    -0.000000484
    -0.000000003
    -0.000000003
    -0.000510625
    -0.000510645
    -0.000020338
    -0.000020323
    -0.000980719
    -0.000980712
    -0.000342417
    -0.000342384
    -0.000338354
    -0.000338321
    -0.000748767
    -0.000748749
    -0.000892147
    -0.000000072
    -0.000000043
    -0.001114756
    -0.000319178
    -0.000417690
    -0.000417729
    -0.000074022
    -0.000074002
     0.000395046
     0.000491374
     0.000491475
     0.000342455
    -0.000015743
     0.000540550
     0.001846365
     0.000540554
     0.001846378
     0.001614858
     0.001614868
     0.001184133
     0.001504162
     0.001184132
     0.001504168
     0.001133878
     0.001133883
     0.000683765
     0.002058131
     0.000684180
     0.002058140
     0.001895554
     0.001895559
     0.001621671
     0.001621680
     0.001499999
     0.001500008
    -0.000000021
     0.001954396
    -0.000000021
     0.001954412
     0.001803945
     0.001803958
     0.000226377
     0.001676716
     0.000226351
     0.001676726
     0.001443881
     0.001443888
     0.001362303
     0.001876542
     0.001362289
     0.001876554
     0.001552392
     0.001552402
     0.001103163
     0.001697739
     0.001103153
     0.001697749
     0.001396053
     0.001396060
     0.000682786
     0.002005921
     0.000682911
     0.002005935
     0.001791177
     0.001791188
    -0.000009300
     0.001765082
    -0.000009307
     0.001765094
     0.001592025
     0.001592035
     0.000849648
     0.000894049
     0.000849648
     0.000894050
    -0.000126467
     0.000375981
    -0.000126467
     0.000375981
     0.000954467
     0.001448727
     0.000955014
     0.001448711
    -0.000062026
     0.001374541
    -0.000062102
     0.001374603
     0.001016783
     0.001016785
     0.000993634
     0.000993636
     0.001105968
     0.001255426
     0.001105969
     0.001255430
    -0.000002784
     0.000805913
    -0.000002783
     0.000805914
     0.000303734
     0.001915104
     0.000304423
     0.001915103
     0.001834185
     0.001834180
     0.001395798
     0.001395804
     0.001326213
     0.001326219
     0.002017561
     0.002017577
     0.001963867
     0.001963883
     0.000000718
     0.001583088
     0.000000707
     0.001583092
     0.001349370
     0.001349372
     0.001686733
     0.001849665
     0.001686734
     0.001849674
    -0.000029568
     0.001306058
    -0.000029594
     0.001306071
     0.001849165
     0.001849165
     0.001332084
     0.001332091
     0.001014342
     0.001177323
    -0.000000036
     0.000786958
     0.001306323
     0.001227658
     0.001159419
     0.001159422
     0.000141453
     0.000141448
     0.001786167
     0.001786134
     0.000989218
     0.000989612
     0.002099339
     0.002099353
     0.001835600
     0.001835615
     0.000854486
     0.001510208
     0.001237429
    -0.000011829
     0.001570114
     0.001378969
     0.001617712
     0.001617718
     0.000529616
     0.000529636
     0.001905943
     0.001905943
     0.001032406
     0.001032571
     0.001663743
     0.001997045
     0.001663716
     0.001997054
     0.001695177
     0.001695181
     0.001415916
     0.001688773
     0.001415916
     0.001688782
    -0.000000011
     0.001235907
    -0.000000011
     0.001235915
    -0.000018160
     0.001616526
    -0.000018159
     0.001616533
     0.001430999
     0.001431005
     0.001826921
     0.001826936
     0.001731725
     0.001731738
     0.000926825
     0.001555124
     0.000926817
     0.001555130
     0.001275920
     0.001275924
     0.002086993
     0.002087008
     0.001976710
     0.001976722
     0.001343787
     0.001446960
     0.001343791
     0.001446966
    -0.000113954
     0.000808955
    -0.000113949
     0.000808957
     0.001862993
     0.001982411
     0.001862964
     0.001982410
    -0.000005162
     0.001734854
    -0.000005173
     0.001734862
     0.001055804
     0.001394214
     0.001055803
     0.001394217
     0.001069001
     0.001069003
     0.002072683
     0.002072693
     0.002019591
     0.002019598
     0.001501935
     0.001501938
     0.001366193
     0.001366196
     0.001618738
     0.001618749
     0.001575299
     0.001575310
