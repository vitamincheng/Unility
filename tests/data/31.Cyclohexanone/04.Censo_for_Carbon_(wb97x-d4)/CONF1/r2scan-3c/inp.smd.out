----------------------------------------------- 
  Element       SASA      Sigma Z        Total  
     Z         Ang**2 cal/Ang**2/mol   kcal/mol 
----------------------------------------------- 
      H        118.372       0.000       0.000
      C         26.633      91.879       2.447
      O         25.172       2.693       0.068

 Subtotal:     170.177                   2.515
 -----------------------------------------------

  Elements    COT*SASA   Sigma Z,Z'     Total   
    Z,Z'       Ang**2  cal/Ang**2/mol kcal/mol  
 ---------------------------------------------- 
  H   C         79.682     -52.587      -4.190
  H   O          0.000     -28.036      -0.000
  C   C         32.794     -89.718      -2.942
  O   C         16.152      -8.302      -0.134

 Subtotal:                              -7.267
 ---------------------------------------------- 

 NONAQUEOUS SOLVENT CONTRIBUTION (kcal/mol):
 (proportional to SASA, atomic number independent)
 ---------------------------------------------- 
 Subtotal:                               1.647
 ---------------------------------------------- 

 CDS TOTAL (kcal/mol):                  -3.105
...............................................................................
